src/components/form/FormCompanyChallenge.vue

```diff
@@ -0,0 +1,270 @@
+ * - `FormFieldTextRequired`: Component to render required text field.
+ *
+ * @example
+ * <form-company-challenge>
+ *
+ * @see [Figma Design](https://www.figma.com/design/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?node-id=4858-106325&t=MqCoIBTXNV4xkXVk-1)
+ */
+
+// libraries
+import { defineComponent, ref } from 'vue';
+
+// components
+import FormFieldDateRequired from '../form/FormFieldDateRequired.vue';
+import FormFieldTextRequired from '../global/FormFieldTextRequired.vue';
+
+// types
+import type { TransportType } from '../types/Route';
+type ChallengeType = 'regularity' | 'performance';
+type ChallengeParticipants = 'individuals' | 'teams' | 'subsidiaries';
+
+export default defineComponent({
+  name: 'FormCompanyChallenge',
+  components: {
+    FormFieldDateRequired,
+    FormFieldTextRequired,
+  },
+  setup() {
+    const challengeType = ref<ChallengeType>('regularity');
+    const challengeParticipants = ref<ChallengeParticipants>('individuals');
+    const challengeTransportType = ref<TransportType[]>(['bike', 'walk']);
+    const challengeTitle = ref<string>('');
+    const challengeDescription = ref<string>('');
+    const challengeInfoUrl = ref<string>('');
+    const challengeStart = ref<string>('');
+    const challengeStop = ref<string>('');
+
+    return {
+      challengeType,
+      challengeParticipants,
+      challengeTransportType,
+      challengeTitle,
+      challengeDescription,
+      challengeInfoUrl,
+      challengeStart,
+      challengeStop,
+    };
+  },
+});
+</script>
+
+<template>
+  <div data-cy="form-company-challenge">
+    <!-- Section: Challenge type -->
+    <fieldset data-cy="form-challenge-type" class="q-pa-none no-border">
+      <legend class="text-grey-10 text-caption text-bold q-mb-sm">
+        {{ $t('form.labelChallengeType') }}
+      </legend>
+      <div class="q-gutter-sm">
+        <!-- Regularity -->
+        <q-radio
+          v-model="challengeType"
+          val="regularity"
+          :label="$t('form.labelChallengeTypeRegularity')"
+          data-cy="form-challenge-type-regularity"
+        />
+        <!-- Performance -->
+        <q-radio
+          v-model="challengeType"
+          val="performance"
+          :label="$t('form.labelChallengeTypePerformance')"
+          data-cy="form-challenge-type-performance"
+        />
+      </div>
+    </fieldset>
+    <!-- Section: Participants -->
+    <fieldset
+      class="q-mt-lg q-pa-none no-border"
+      data-cy="form-challenge-participants"
+    >
+      <legend class="text-grey-10 text-caption text-bold q-mb-sm">
+        {{ $t('form.labelParticipants') }}
+      </legend>
+      <div class="q-gutter-sm">
+        <!-- Individuals -->
+        <q-radio
+          v-model="challengeParticipants"
+          val="individuals"
+          data-cy="form-participants-individuals"
+        >
+          <q-icon name="person" class="q-mr-xs text-grey-6" size="18px" />
```

Comment:
Icon size repeat > 1, could you set variable (constant) for this value, please?

Reply:
Updated.